website/src/pages/HistoryPage/HistoryPage.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2023 The Vitess Authors.
```

Comment:
nit: when creating new files, we should update the license header to the current year.

```suggestion
Copyright 2024 The Vitess Authors.
```